ios/StripeTerminalReactNative.swift

```diff
@@ -513,6 +525,18 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
         sendEvent(withName: ReactNativeConstants.CHANGE_CONNECTION_STATUS.rawValue, body: ["result": result])
     }
 
+    func terminal(_ terminal: Terminal, didStartReaderReconnect cancelable: Cancelable) {
```

Comment:
looks like we are not doing much with the cancelable here. We should save that to var like `cancelReaderConnectionCancellable` and then expose a `cancelReaderReconnection` method for users to call
cc @bric-stripe

Reply:
ah, good catch. Yeah that should be made available to the integrations so it can be used 👍